app/src/main/res/values/strings.xml

```diff
@@ -673,41 +673,41 @@
     <string name="dsa_affine_y">Affine y coordinate</string>
     <string name="critical_exts">Critical Extensions</string>
     <string name="non_critical_exts">Non-critical Extensions</string>
-    <string name="usage_access_not_supported">Could not request usage access as the corresponding settings page does not exist.</string>
+    <string name="usage_access_not_supported">Could not request usage access, as the corresponding settings page does not exist.</string>
     <string name="packages">packages</string>
     <string name="onboard">Onboard</string>
     <string name="do_you_want_to_uninstall_and_install">Do you want to uninstall and install the app again?</string>
-    <string name="signature_mismatch_for_system_apps">Could not install app because the installed app is a system app with different signature.</string>
+    <string name="signature_mismatch_for_system_apps">Could not install app because the installed one is a system app with different signature.</string>
     <string name="app_data_will_be_lost">The existing data will be lost.</string>
-    <string name="install_without_data_loss">If you have disabled signature verification, you can use the <b>only install</b> option to install the app without losing data.</string>
+    <string name="install_without_data_loss">If you have turned off signature verification, you can use the <b>Only install</b> option to install the app without losing data.</string>
     <string name="only_install">Only Install</string>
     <string name="sys_config">System Config</string>
     <string name="scanner">Scanner</string>
     <string name="no_libs">No libraries</string>
     <string name="lib_details">Library Details</string>
     <string name="tap_to_see_details">Tap to see details</string>
-    <string name="view_missing_signatures">Tap to view signatures that are not yet present in App Manager\'s database of libraries. You can also send them to me if you want.</string>
+    <string name="view_missing_signatures">Tap to view or send in signatures not yet present in App Manager\'s database of libraries.</string>
```

Comment:
I think the modification has destroyed the original meaning. There's no option to send the signatures in the dialog which will appear after clicking here. However, there is an option to copy the signatures, and the user can send them to me manually via other means. (Probably I should change this as well?)

Reply:
I can leave it like this, and you can add a "send via" dialog?

Reply:
Alright. I'll replace _copy_ with _send_.